### ISSUE TITLE
Accessibility: Missing label associations on serviceProfessionalServiceEditor when adding services/consultations

### ISSUE THREAD

**Issue by ndarilek** [loconomics/loconomics#629]
On the screen to add services, the following fields are not associated with their labels (I.e. no `for` attribute on the label corresponding with an `id` attribute on the input):

 * Price for service
 * Hours
 * Minutes

Likewise, on the screen to add consultations, these fields are unlabeled:

 * Price for consultation
 * Hours
 * Minutes

**Comment by joshdanielson** [comment]
Done as part of #528. Closing.
